README.md

```diff
@@ -1,6 +1,7 @@
 # Zenko Backbeat
 
 ![backbeat logo](res/backbeat-logo.png)
+![Build Status](https://github.com/scality/backbeat/actions/workflows/release.yaml/badge.svg)
```

Comment:
should this not be the badge for the "test" workflow?

Reply:
Not sure it's useful as the `development/x.y` branches dont't run that workflow (only release is run on them), so it would be getting the status from the last run on some dev branch.